scripts/install/setup.sh

```diff
@@ -16,6 +16,21 @@ PARENT_DIR=$PARENT_DIR $PARENT_DIR/spinnaker-for-gcp/scripts/manage/check_git_co
 
 source "$PROPERTIES_FILE"
 
+LOCATION_GKE="--zone $ZONE"
```

Comment:
Thanks for taking this on; several folks have asked for regional cluster support, so it will definitely get used.